_content/devices/bytes.md

```diff
@@ -228,7 +271,34 @@ float myVal = payload[0] / 100.00;
 
 ## How to send multiple numbers?
 
-In a lot of cases you'll want to send multiple values in a single message. Start off by encoding each individual number to a buffer of bytes and then combine them into a single buffer.
+In a lot of cases you'll want to send multiple values in a single message. Often one simply sees that each byte is explicitly mapped to a specific position in the payload:
+
+Encode (Arduino):
+
+```c
+byte payload[6];
+
+payload[0] = myValA;
+
+payload[1] = highByte(myValB);
+payload[2] = lowByte(myValB);
+
+payload[3] = (byte) ((myValC & 0xFF0000) >> 16);
+payload[4] = (byte) ((myValC & 0x00FF00) >> 8 );
+payload[5] = (byte) ((myValC & 0X0000FF)      );
```

Comment:
See above; I'm fine removing the mask here

Reply:
I agree